src/NServiceBus.Transport.RabbitMQ/Administration/ManagementClient/Converters/DeliveryLimitConverter.cs

```diff
@@ -0,0 +1,34 @@
+﻿#nullable enable
+
+namespace NServiceBus.Transport.RabbitMQ.Administration.ManagementClient.Converters;
```

Comment:
Let's change all of the management client namespaces to 
```suggestion
namespace NServiceBus.Transport.RabbitMQ.ManagementClient;
```